osdep/timer-win32.c

```diff
@@ -115,14 +130,14 @@ void mp_raw_time_init(void)
     if ((v = getenv("MPV_HRT_MAX"))) {
         int hmax = atoi(v);
         if (hmax >= 1 && hmax <= 1000)
-            hires_max = hmax;
+            hires_max = MP_TIME_MS_TO_NS(hmax);
     }
 
     // 1..15 ms hires resolution (not used in "never" mode)
     if ((v = getenv("MPV_HRT_RES"))) {
         int res = atoi(v);
         if (res >= 1 && res <= 15)
-            hires_res = res;
+            hires_res = MP_TIME_MS_TO_NS(res);
```

Comment:
We should change `MPV_HRT_RES` not to at lest microseconds, so we also set 0.5 ms or any other value that may be supported